VK/VK/Network/NetworkManager.swift

```diff
@@ -0,0 +1,78 @@
+// NetworkManager.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+// Сетевой слой
+final class NetworkManager {
+    // MARK: - Private Properties
+
+    private let configuration = URLSessionConfiguration.default
+
+    // MARK: - Public Methods
+
+    func getFriends() {
+        let session = URLSession(configuration: configuration)
+        guard let url = URL(
```

Comment:
```suggestion
        guard let url = URL(
```
надо через URLComponents